librz/util/file.c

```diff
@@ -1181,6 +1181,32 @@ RZ_API bool rz_file_copy(const char *src, const char *dst) {
 	/* TODO: Use NO_CACHE for iOS dyldcache copying */
 #if HAVE_COPYFILE
 	return copyfile(src, dst, 0, COPYFILE_DATA | COPYFILE_XATTR) != -1;
+#elif HAVE_COPY_FILE_RANGE && !defined(__FreeBSD__)
+	/**
+	 * Unfortunately on FreeBSD the syscall
+	 * does not live up to the promise
+	 */
```

Comment:
can we set `HAVE_COPY_FILE_RANGE` to false when the system is freebsd instead of this? I think it makes more sense to have all the build configuration stuff in meson instead of in the code.